app/views/state_file/questions/nj_tenant_eligibility/edit.html.erb

```diff
@@ -0,0 +1,44 @@
+<%
+    title = t(".title", filing_year: @filing_year)
+    content_for :page_title, title
+%>
+
+<% content_for :card do %>
+  <%= form_with model: @form, url: { action: :update }, local: true, method: "put", builder: VitaMinFormBuilder, html: { class: "form-card form-card--long" } do |f| %>
+    <h1 class="h2"><%= title %></h1>
+    <p><%= t(".label") %></p>
```

Comment:
[pebble] I know we still need to go back and fix the individual checkboxes being in fieldsets thing with CfA, and I  added that to the sundry a11y issues ticket. In the mean time we can get closer to level A WCAG compliance by programmatically associating the question and the options here with a fieldset and legend rather than a div and a p tag.